client/app/components/index.jsx

```diff
@@ -36,10 +38,29 @@ class App extends React.Component {
     });
   }
 
-  setDirections(directions) {
-   this.setState({
-     directions: directions
-   });
+  setMap(map) {
+    if (!this.state.map) {
+      this.setState({
+        map: map
+      });
+    }
+  }
+
+  getCrimeData(steps) {
+    let coords = steps.map((step) => {
+      return [[step.start_location.lng(), step.start_location.lat()], [step.end_location.lng(), step.end_location.lat()]]
+    });
+    //function to send POST request to server once server-side routes are written
+    // axios.post('/ratings', {streets: coords})
+    //   .then((res) => {
+    //     // this.setState({
+    //     //   streetLines: res
+    //     // })
+    //     console.log(res);
+    //   })
+    //   .catch((err) => {
+    //     console.error(err);
+    //   })
```

Comment:
we can move this axios call (and the others) to another file to make this section a bit less cluttered. not a must do, but something that could be added to the backlog